tasks/move-granules/index.js

```diff
@@ -321,54 +283,45 @@ async function updateEachCmrFileAccessURLs(cmrFiles, granulesObject, distEndpoin
  * @param {Object} event - Lambda function payload
  * @param {Object} event.config - the config object
  * @param {string} event.config.bucket - Bucket name where public/private keys are stored
- * @param {string} event.config.granuleIdExtraction - regex needed to extract granuleId
- *                                                    from filenames
- * @param {Array} event.config.input_granules - an array of granules
+ * @param {Object} event.config.buckets - Buckets config
  * @param {string} event.config.distribution_endpoint - distribution endpoint for the api
  * @param {Object} event.config.collection - collection configuration
  *                     https://nasa.github.io/cumulus/docs/data-cookbooks/setup#collections
  * @param {boolean} [event.config.moveStagedFiles=true] - set to false to skip moving files
  *                                 from staging to final bucket. Mostly useful for testing.
- * @param {Array} event.input - an array of s3 uris
+ * @param {Object} event.input - an array of s3 uris
```

Comment:
This is no longer an array of s3 URIs, correct?

Reply:
correct. a67c76759